### DISCUSSION TITLE
127.0.0.1:8000/press/bank/<page>/<button> not working anymore

### DISCUSSION THREAD

**Discussion by Gerrie12345** [bitfocus/companion#2790] [Q&A] [answered]
I have been using the script below in vMix to remotely control Companion. However, it has stopped working since I updated to V3.2

Dim client As WebRequest = WebRequest.Create("http://127.0.0.1:8000/press/bank/1/1" )
Dim response As WebResponse = client.GetResponse()

I suspect that the 'press bank' command has been changed.

Could you please advise on the updated script I should use now?

**Comment by Julusian** [comment]
There is a new api you could use, but the old api does still work. It is disabled by default in new installations of companion, so check the companion settings

**Comment by Gerrie12345** [comment, marked as answer]
Thanks, I enabled 'Deprecated HTTP API' and it works again.

ChatGPT wrote this for me for the new api:

Dim request As HttpWebRequest = DirectCast(WebRequest.Create("http://127.0.0.1:8000/api/location/1/1/1/press"), HttpWebRequest)
request.Method = "POST"

Dim response As String = New StreamReader(request.GetResponse().GetResponseStream()).ReadToEnd()
Console.WriteLine("API Response: " & response)

It works, but is this the best way to do it?

**Reply by Julusian** [reply]
I don't know c# well enough to answer that
